Core/RoverControllerPresets/DriveControllers/ForzaLikeController.cs

```diff
@@ -18,15 +20,12 @@ public RoverControl CalculateMoveVector()
 
 		turn *= velocity; // Max turn angle: 45 deg.
 
-		Vector2 vec = new Vector2(velocity, turn);
+		Vector3 vec = new Vector3(velocity, turn, 0);
 		float forcedSteer = Input.GetAxis("rover_rotate_right", "rover_rotate_left");
 
 		if (!Mathf.IsEqualApprox(forcedSteer, 0f, 0.05f))
 			vec.Y = forcedSteer / 5f;
 
-		if (Input.IsActionPressed("camera_zoom_mod"))
-			vec /= 8f;
-
-		return RoverControlVec2Extension.FromVector2(vec);
+		return RoverControlVec3Extension.ToRoverControl(vec);
```

Comment:
Prefer using extension methods from extended classes.
```suggestion
		return vec.ToRoverControl();
```
see:
https://github.com/wisniax/RoverControlApp/blob/3b3e3cfab920852433be6b94e847b16dd4c70398/Core/RoverControllerPresets/MqttClasses.RoverControlVec3Extension.cs#L12-L13

Reply:
Adressed in [5ed280b](https://github.com/wisniax/RoverControlApp/pull/36/commits/5ed280b9da4555ee40b428e3d684a0ecffc34549)